pkg/agent/openflow/client.go

```diff
@@ -224,13 +227,32 @@ func (c *client) InstallPodFlows(containerID string, podInterfaceIP net.IP, podI
 			c.l3ToPodFlow(podInterfaceIP, podInterfaceMAC, cookie.Pod),
 		)
 	}
-	return c.addFlows(c.podFlowCache, containerID, flows)
+	return c.addFlows(c.podFlowCache, interfaceName, flows)
+}
+
+func (c *client) UninstallPodFlows(interfaceName string) error {
+	c.replayMutex.RLock()
+	defer c.replayMutex.RUnlock()
+	return c.deleteFlows(c.podFlowCache, interfaceName)
 }
 
-func (c *client) UninstallPodFlows(containerID string) error {
+func (c *client) GetPodFlowKeys(interfaceName string) []string {
+	fCacheI, ok := c.podFlowCache.Load(interfaceName)
+	if !ok {
+		return nil
+	}
+
+	fCache := fCacheI.(flowCache)
+	flowKeys := make([]string, 0, len(fCache))
+	// ReplayFlows() could change Flow internal state. Although its current
+	// implementation does not impact Flow match string generation, we still
+	// acquire read lock of replayMutex here for logic cleaness.
```

Comment:
I think "cleaness" is not a word. cleanliness is.

Reply:
Thanks. Fixed it.